src/vma/util/lock_wrapper.h

```diff
@@ -68,7 +68,7 @@ class lock_base
 #endif
 protected:
 	tscval_t                start_lock_wait() { return 0; };
-	void                    end_lock_wait(tscval_t start_time) {start_time = 0; NOT_IN_USE(start_time);};
+	void                    end_lock_wait(tscval_t start_time) {start_time = 0; start_time = start_time;};
```

Comment:
I don't feel comfortable pushing this code without further understanding (research and testing).

@igor-ivanov , can you separate this to different PR so we can further discuss. I don't want this to block everything else.